src/routes.py

```diff
@@ -86,3 +86,17 @@ def topics() -> str | Response:
         topic_list = session.query(Topic).order_by(Topic.created_at.desc()).all()
         return render_template("topics.html", topic_list=topic_list, current_user=current_user)
     return redirect(url_for("routes.login"))
+
+
+@bp.route("/topics/create")
+def create_topic() -> str:
+    """Hadle create topic page."""
+    session_id = request.cookies.get("session_id")
+    user_session = session.query(UserSession).filter_by(session_id=session_id).first()
+
+    current_user = None
+    if user_session is not None:
+        current_user = session.query(User).filter_by(id=user_session.user_id).one()
+
+    form = TopicForm()
+    return render_template("topics-create.html", form=form, current_user=current_user)
```

Comment:
I can see a topic creation page as a non signed in user. Let's add a redirect to the login page if anonymous user tries to access topic creation page.

Reply:
But yeah, since topic creation functionality isn't actually working right now - I mean even if anonymous user has access to this page he cannot create topics actually. So It's OK if we will add this redirect in another ticket - when we will implement functionality for this page. 

This change is not required now. Close as resolved.